templates/vue/src/components/NodeModal.vue

```diff
@@ -805,5 +914,9 @@ table {
       margin-bottom: 0;
     }
   }
+
+  #node-shapeship-coordinate-text {
+    padding-top: 6px;
+  }
```

Comment:
Please use bootstrap classes for minor spacing adjustments instead of your own CSS. See https://getbootstrap.com/docs/4.0/utilities/spacing/ (e.g. `pt-2` or something like that)

Reply:
(also you've misspelled spaceship here it says "shapeship")